app/classes/amity.py

```diff
@@ -142,31 +146,29 @@ def load_people(self, filename):
 				if role == 'STAFF':
```

Comment:
Do we need to check the role of the person here again if our add_person function is also doing the same?